applications/concept-catalogue/src/main/kotlin/no/begrepskatalog/conceptregistration/validation/BegrepValidator.kt

```diff
@@ -0,0 +1,76 @@
+package no.begrepskatalog.conceptregistration.validation
+
+import no.begrepskatalog.generated.model.Begrep
+import no.begrepskatalog.generated.model.Kildebeskrivelse
+import no.begrepskatalog.generated.model.Status
+import no.begrepskatalog.generated.model.URITekst
+import org.hibernate.validator.internal.util.ModUtil
+
+class BegrepValidator {

```

Comment:
Du kan dra hele validate metoden ut av klassen, virker unødvendig å ha en klasse bare for companion object